### PR TITLE
Renaming easyTable to mobileTable.

### PR DESCRIPTION
**Pull request by brettsam**
…The comprehensive rename is coming later.

This unblocks templates and docs to start using `mobileTable` and I can come back to fix up all the Extensions code later (within a week, I'd say).

We're currently discussing on the side how we roll out this breaking change -- it'll break anyone using an `easyTable` binding. It's easy enough to support both binding type names, but we may want to use this to try out versioning support.

Edit: This now includes the comprehensive rename.

### CODE REVIEW

File: sample/EasyTables-CSharp/function.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
     {
       "type": "queueTrigger",
```
Comment (mathewc) [inline sample/EasyTables-CSharp/function.json]: Rename the file folders as well

File: src/WebJobs.Script/Binding/FunctionBinding.cs [hunk 2]
```diff
@@ -100,7 +100,7 @@ internal static Collection<FunctionBinding> GetBindings(ScriptHostConfiguration
                         case BindingType.HttpTrigger:
                             bindings.Add(new HttpBinding(config, bindingMetadata, FileAccess.Read));
                             break;
-                        case BindingType.EasyTable:
+                        case BindingType.MobileTable:
                             EasyTableBindingMetadata easyTableMetadata = (EasyTableBindingMetadata)bindingMetadata;
```
Comment (mathewc) [inline src/WebJobs.Script/Binding/FunctionBinding.cs]: Why not just go all the way in the Script repo and rename all the artifacts in here, including the binding classes, etc.? Heck, I don't know why you don't just rename the extension now, pull that in and do it all in one go. Should take about an hour.

**Comment by paulbatum** [comment]
Can we put the version number bump (0.2.x) in this PR so that we make sure it doesn't get forgotten?

File: WebJobs.Script.proj [hunk 3]
```diff
@@ -7,7 +7,7 @@
     <SkipStrongNamesXml>tools\SkipStrongNames.xml</SkipStrongNamesXml>
     <PublishPath Condition=" '$(PublishPath)' == '' ">bin</PublishPath>
     <BuildNumber Condition=" '$(BuildNumber)' == '' ">0</BuildNumber>
-    <SiteExtensionVersion>0.1.$(BuildNumber)</SiteExtensionVersion>
+    <SiteExtensionVersion>0.2.$(BuildNumber)</SiteExtensionVersion>
```
Comment (brettsam) [inline WebJobs.Script.proj]: This looks like it worked -- @mathewc is there anywhere else that I need to update to bump up the version?

**Comment by brettsam** [comment]
@paulbatum -- I've re-done the commit to be a full rename with the new Extensions package. I've also included the version bump.

File: WebJobs.Script.proj (hunk 3 above)
Comment (fabiocav) [inline WebJobs.Script.proj]: This should be it. If the package zip and the folder in it have the appropriate version, you should be good.

Comment (mathewc) [inline WebJobs.Script.proj]: Yeah, I think that's it

File: src/WebJobs.Script.Host/WebJobs.Script.Host.csproj [hunk 4]
```diff
@@ -95,8 +95,8 @@
       <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.DocumentDB.1.0.0-alpha-10268\lib\net45\Microsoft.Azure.WebJobs.Extensions.DocumentDB.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.Azure.WebJobs.Extensions.EasyTables, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.EasyTables.1.0.0-alpha-10268\lib\net45\Microsoft.Azure.WebJobs.Extensions.EasyTables.dll</HintPath>
+    <Reference Include="Microsoft.Azure.WebJobs.Extensions.MobileApps, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (mathewc) [inline src/WebJobs.Script.Host/WebJobs.Script.Host.csproj]: What do you think we should do with the old package - remove it from myget once these updates are in place?

**Comment by mathewc** [comment]
:ship:

File: src/WebJobs.Script.Host/WebJobs.Script.Host.csproj (hunk 4 above)
Comment (brettsam) [inline src/WebJobs.Script.Host/WebJobs.Script.Host.csproj]: Yeah, we should just delete it.

**Pull request merged**